R/rt_taxo_ranks.R

```diff
@@ -1,28 +1,22 @@
-#' Retrieve the list of taxonomic ranks in TaxRef
+#' Retrieve taxonomic ranks in TaxRef
 #'
-#' @export
-rt_taxo_ranks = function() {
-
-  api_query = rt_GET("taxonomicRanks")
-
-  parse_taxa(api_query)
-}
-
-#' Retrieve a taxonomic rank used in TaxRef
+#' If the function is used without arguments returns the entire list of
+#' taxonomic ranks used in TaxRef.
 #'
-#' @param taxo_id {`character(1)`}\cr{}
-#'                the id of a taxonomic rank in TaxRef see
-#'                [`rt_taxo_ranks()`]
+#' @param taxo_id {`character(1)`, default = `NULL`}\cr{}
+#'                the id of a taxonomic rank in TaxRef
 #'
 #' @examples
-#' rt_taxo_ranks_id(taxo_id = "VAR")
+#' rt_taxo_ranks(taxo_id = "VAR")
```

Comment:
Same comment about the case and  `tolower()`

Reply:
In this case the arguments can be mixed case so it is impossible to use `tolower()` nor `toupper()` (the only exception being the `Dumm`  case)